### ISSUE TITLE
This is the only error output.

### ISSUE THREAD

**Issue by tdcarr** [gmetais/YellowLabTools#199]
www.google.fr runs the test fine.
Any other sites cause the following error:
Sending response without waiting.
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: write EPIPE
    at exports._errnoException (util.js:907:11)
    at WriteWrap.afterWrite (net.js:785:14)

CentOS 7.
Node 4.5.0
YLT is latest.

**Comment by gmetais** [comment]
Hi Terry,

Thanks for reporting this issue. EPIPE means that the other part closed the connection before the end of the response could be sent.

Question 1: Are you using the web interface or the API?

Question 2: How long between these two lines?
```
Sending response without waiting.
events.js:141
```

**Comment by tdcarr** [comment]
Question 1: Web Interface.

Question 2: About 45 seconds.

**Comment by gmetais** [comment]
It could be a bug with imagemin. Could you please try to delete the followling modules in your node_modules directory and reinstall them?

- imagemin
- imagemin-jpegoptim
- imagemin-jpegtran
- imagemin-optipng
- imagemin-svgo

Watch carefully if there's any error during their installation.

**Comment by FynnZW** [comment]
I had the same problem on: 

- OSX Sierra
- npm 4.6.1
- node 5.12.0

I went to `/usr/local/lib/node_modules/yellowlabtools`, deleted the imagemin-modules and reinstalled them, but it still failed on alle pages except google.com/fr.

I used [npm-check](https://www.npmjs.com/package/npm-check) to update all minor and patch updates. It work's now!
Unfortunately, I don'd know which package update did the trick, but here's the new (working) package.json to compare:

```json
{
  "_args": [
    [
      "yellowlabtools",
      ""
    ]
  ],
  "_from": "yellowlabtools@latest",
  "_id": "yellowlabtools@1.12.3",
  "_inCache": true,
  "_installable": true,
  "_location": "/yellowlabtools",
  "_nodeVersion": "4.4.5",
  "_npmOperationalInternal": {
    "host": "s3://npm-registry-packages",
    "tmp": "tmp/yellowlabtools-1.12.3.tgz_1495965028605_0.06616444839164615"
  },
  "_npmUser": {
    "email": "gael.metais@gmail.com",
    "name": "gmetais"
  },
  "_npmVersion": "2.15.5",
  "_phantomChildren": {},
  "_requested": {
    "name": "yellowlabtools",
    "raw": "yellowlabtools",
    "rawSpec": "",
    "scope": null,
    "spec": "latest",
    "type": "tag"
  },
  "_requiredBy": [
    "#USER"
  ],
  "_resolved": "https://registry.npmjs.org/yellowlabtools/-/yellowlabtools-1.12.3.tgz",
  "_shasum": "d6666bdc2a625293b732cc2ff8e8656a4ad7a622",
  "_shrinkwrap": null,
  "_spec": "yellowlabtools",
  "_where": "",
  "author": {
    "email": "gael@gaelmetais.com",
    "name": "Gaël Métais",
    "url": "http://www.gaelmetais.com"
  },
  "bin": {
    "yellowlabtools": "./bin/cli.js"
  },
  "bugs": {
    "url": "https://github.com/gmetais/YellowLabTools/issues"
  },
  "dependencies": {
    "angular": "^1.6.4",
    "angular-animate": "^1.6.4",
    "angular-chart.js": "1.1.1",
    "angular-local-storage": "0.5.0",
    "angular-resource": "^1.6.4",
    "angular-route": "^1.6.4",
    "angular-sanitize": "^1.6.4",
    "async": "1.5.2",
    "body-parser": "^1.17.2",
    "chart.js": "^2.6.0",
    "clean-css": "3.4.23",
    "color-diff": "^1.1.0",
    "compression": "1.6.2",
    "cors": "^2.8.3",
    "css-mq-parser": "0.0.3",
    "debug": "^2.6.8",
    "easyxml": "2.0.1",
    "ejs": "^2.5.6",
    "express": "^4.15.3",
    "fontkit": "^1.6.1",
    "imagemin": "^5.3.1",
    "imagemin-jpegoptim": "^5.1.0",
    "imagemin-jpegtran": "5.0.2",
    "imagemin-optipng": "5.2.1",
    "imagemin-svgo": "^5.2.2",
    "is-eot": "1.0.0",
    "is-gif": "1.0.0",
    "is-http2": "1.1.0",
    "is-jpg": "1.0.0",
    "is-otf": "0.1.2",
    "is-png": "^1.1.0",
    "is-svg": "2.1.0",
    "is-ttf": "0.2.2",
    "is-woff": "1.0.3",
    "is-woff2": "1.0.0",
    "lwip": "0.0.9",
    "md5": "2.2.1",
    "meow": "3.7.0",
    "minimize": "^2.1.0",
    "parse-color": "1.0.0",
    "phantomas": "1.19.0",
    "ps-node": "0.1.4",
    "q": "^1.5.0",
    "request": "^2.81.0",
    "rimraf": "^2.6.1",
    "temporary": "0.0.8",
    "try-thread-sleep": "^1.0.2",
    "uglify-js": "2.7.5"
  },
  "description": "Online tool to audit a webpage for performance and front-end quality issues",
  "devDependencies": {
    "chai": "~3.5.0",
    "grunt": "~1.0.1",
    "grunt-blanket": "~0.0.10",
    "grunt-contrib-clean": "^1.1.0",
    "grunt-contrib-concat": "~1.0.1",
    "grunt-contrib-copy": "~1.0.0",
    "grunt-contrib-cssmin": "~1.0.2",
    "grunt-contrib-htmlmin": "^2.4.0",
    "grunt-contrib-jshint": "~1.1.0",
    "grunt-contrib-less": "~1.4.0",
    "grunt-contrib-uglify": "~2.0.0",
    "grunt-env": "~0.4.4",
    "grunt-express": "~1.4.1",
    "grunt-filerev": "~2.3.1",
    "grunt-inline-angular-templates": "~0.1.5",
    "grunt-line-remover": "~0.0.2",
    "grunt-mocha-test": "~0.13.2",
    "grunt-replace": "~1.0.1",
    "grunt-usemin": "~3.1.1",
    "grunt-webfont": "^1.6.0",
    "matchdep": "~1.0.1",
    "mocha": "^3.4.2",
    "sinon": "~1.17.4",
    "sinon-chai": "^2.10.0"
  },
  "directories": {},
  "dist": {
    "shasum": "d6666bdc2a625293b732cc2ff8e8656a4ad7a622",
    "tarball": "https://registry.npmjs.org/yellowlabtools/-/yellowlabtools-1.12.3.tgz"
  },
  "engines": {
    "node": ">= 4.0"
  },
  "gitHead": "7f687051f8fda9c79d936f2c9519c48a24e21bb7",
  "homepage": "https://github.com/gmetais/YellowLabTools#readme",
  "keywords": [
    "performance",
    "metrics",
    "webperf",
    "pagespeed",
    "budget",
    "phantomas"
  ],
  "license": "GPL-2.0",
  "main": "./lib/index.js",
  "maintainers": [
    {
      "email": "gael.metais@gmail.com",
      "name": "gmetais"
    }
  ],
  "name": "yellowlabtools",
  "optionalDependencies": {},
  "readme": "ERROR: No README data found!",
  "repository": {
    "type": "git",
    "url": "git://github.com/gmetais/YellowLabTools.git"
  },
  "scripts": {
    "test": "grunt test"
  },
  "version": "1.12.3"
}

```

**Comment by gmetais** [comment]
Should be fixed in v1.12.4 (I updated almost every outdated packages). Don't hesitate to answer if the bug is still there.
